libs/ngx-dsfr/upload/src/lib/testing/upload.component.spec.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Angular imports
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { FormsModule } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { RouterLinkDirectiveStub } from '@betagouv/ngx-dsfr/testing';
```

Comment:
You can remove the imports for `FormsModule` and `RouterLinkDirectiveStub` ( and the "3rd-party imports" comment )